### ISSUE TITLE
Possible to add console prints to executeAnalysis?

### ISSUE THREAD

**Issue by mdlavallee92** [OHDSI/CohortIncidence#41]
When I run `CohortIncidence::executeAnalysis` , three scripts are executed: 1) Temp DDL for incidence, 2) the incidence analysis sql itself and 3) clean up tasks. 

Is it possible to add some sort of console print that distinguishes what is going on with each sql execution? This is not a big deal in terms of the execution but as a user I would like to know what is being read/written to the database as some sort of transaction is taking place.

From [here](https://github.com/OHDSI/CohortIncidence/blob/master/R/CohortIncidence.R#L119-L137), something like:

``` r
tempDDL <- SqlRender::translate(CohortIncidence::getResultsDdl(useTempTables = T), 
        targetDialect = targetDialect)
cli::cat_bullet("Build temporary DDL for Incidence Analysis", bullet = "pointer", bullet_col = "yellow")  # console print 
DatabaseConnector::executeSql(conn, tempDDL)


analysisSql <- CohortIncidence::buildQuery(incidenceDesign = irDesign, 
        buildOptions = buildOptions)
analysisSql <- SqlRender::translate(analysisSql, targetDialect = targetDialect)
analysisSqlQuries <- SqlRender::splitSql(analysisSql)
cli::cat_bullet("Executing Incidence Analysis", bullet = "pointer", bullet_col = "yellow") # console print 
DatabaseConnector::executeSql(conn, analysisSql)
results <- DatabaseConnector::querySql(conn, SqlRender::translate("select * from #incidence_summary", 
        targetDialect = targetDialect))


cleanupSql <- SqlRender::translate(CohortIncidence::getCleanupSql(useTempTables = T), 
        targetDialect)
cli::cat_bullet("Clean-up", bullet = "pointer", bullet_col = "yellow")  # console print 
DatabaseConnector::executeSql(conn, cleanupSql)

```

**Comment by chrisknoll** [comment]
Sounds reasonable, but I think we log/report these things through parrallel logger, yes?  I'm sorry to say i'm not sure what the 'preferred' approach for reporting info to console is.

@anthonysena or @schuemie , do you have suggestions?

**Comment by mdlavallee92** [comment]
`ParallelLogger` is fine by me. Sometimes it has issues when used within [R markdown](https://stackoverflow.com/questions/73135786/issues-when-using-logger-package-in-r-markdown-when-initializing-log-errors) and I think `targets` (I remember getting weird error messages from handlers but could have been from something else). 

I like `cli` because its a little prettier but not married to that suggestion, I will use whatever the HADES recommendation is.

**Comment by chrisknoll** [comment]
Ok, then. We have our answer. I'll approve the PR.

**Comment by chrisknoll** [comment]
Fixed in #44
